ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-ec.yaml

```diff
@@ -89,19 +89,7 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
-    - as: set-upgrade-releases
-      commands: echo "${RELEASE_IMAGE_TARGET},${RELEASE_IMAGE_LATEST}" > ${SHARED_DIR}/upgrade-edge
-      dependencies:
-      - env: RELEASE_IMAGE_TARGET
-        name: release:target
-      - env: RELEASE_IMAGE_LATEST
-        name: release:latest
-      from: cli
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
-    - ref: cucushift-chainupgrade-toimage
+    - chain: openshift-upgrade-qe-sanity-rollback-stable
```

Comment:
do we intently to drop the support of `rollback` in nightly job?

Reply:
`rollback` cli doesn't work for nightly because it doesn't accept any options, even `--force`

Reply:
do u mean `nightly` build is not signed and `rollback` cli does not support `--force` option, so it doesn't work for nightly, but `rollback` command would not prevent it against a cluster with nightly in `cv.history` from oc cli, right?

Reply:
Right, `oc` client will not error out, instead, it will send correct rollback request. But the rollback will be stuck on the signature verification as `force: false`